addons/html_editor/static/src/core/selection_plugin.js

```diff
@@ -157,7 +158,14 @@ export class SelectionPlugin extends Plugin {
 
     setup() {
         this.resetSelection();
-        this.addDomListener(this.document, "selectionchange", this.updateActiveSelection);
+        this.addDomListener(this.document, "selectionchange", () => {
+            this.updateActiveSelection();
+            const selection = this.getEditableSelection();
+            if (!selection.isCollapsed || !selection.focusNode) {
```

Comment:
This will affect the behavior outside of the editor too, which is something we never want. You should check that the selection is in the editable. There is a property for that.